site/content/docs/5.0/examples/heroes/index.html

```diff
@@ -100,9 +100,9 @@ <h1 class="display-4 fw-bold lh-1">Border hero with cropped image and shadows</h
           <button type="button" class="btn btn-outline-secondary btn-lg px-4">Default</button>
         </div>
       </div>
-      <div class="col-lg-4 offset-lg-1 p-0 position-relative overflow-hidden shadow-lg">
-        <div class="position-lg-absolute top-0 left-0 overflow-hidden">
-          <img class="d-block rounded-lg-3" src="bootstrap-docs.png" alt="" width="720">
+      <div class="col-lg-4 offset-lg-1 p-0 overflow-hidden shadow-lg">
+        <div class="top-0 left-0">
+          <img class="rounded-lg-3" src="bootstrap-docs.png" alt="" width="720">
         </div>
       </div>
```

Comment:
I think this can be reduced down further to remove the inner `<div>`.

```suggestion
      <div class="col-lg-4 offset-lg-1 p-0 overflow-hidden shadow-lg">
          <img class="rounded-lg-3" src="bootstrap-docs.png" alt="" width="720">
      </div>
```